tools/loader/ImageClassifier.cpp

```diff
@@ -299,6 +315,9 @@ int main(int argc, char **argv) {
 
     // Perform the inference execution.
     loader.runInference(ctx);
+    if (convertInAndOutToFp16) {
+      SMVarT->convertToType(ElemKind::FloatTy);
```

Comment:
Shouldn't this be `ElemKind::Float16Ty`?

Reply:
@jfix71 If I understand the code correctly, then everything is fine. We convert the payload of the tensor containing final results from fp16 into fp32 and then use the usual fp32-based function afterwards to print those results. May be a comment before the `if` explaining what's going on would be useful.

Reply:
That is correct.
I've added a comment to explain that. Let me know what you think.

Reply:
Looks good, thanks!